testing/cli/test_sct_run_batch.py

```diff
@@ -87,6 +87,39 @@ def test_only_one_include(tmp_path, dummy_script):
                             '-path-data', str(data), '-path-out', str(out), '-script', dummy_script])
 
 
+def test_directory_inclusion_exclusion():
+    """
+    Test that -include, -include-list, -exclude, and -exclude-list properly filter directories.
```

Comment:
I was unsure if writing this test was worth the effort. It looks a little silly, and the corresponding function felt a little silly to write, too.

Is this overkill? It's hard to sense what's worth writing a test for and what's not...

Reply:
Since this PR originates from a user request on a forum post, I think it would be good to test that specific scenario in this test, possibly with a link back to the issue in a code comment, so that we avoid future regressions. (Future user requests could turn into added cases.)

For the other test cases, they do look like overkill. This is a self-contained function, so as long as the code is reasonable and it's been tested at least once, it shouldn't suddenly break because of unrelated code changes.